package/kedro_viz/models/flowchart.py

```diff
@@ -20,6 +20,9 @@
 
 logger = logging.getLogger(__name__)
 
+PREVIEW_DATASETS = ["pandas.csv_dataset.CSVDataSet",
+                    "pandas.parquet_dataset.ParquetDataSet", "pandas.excel_dataset.ExcelDataSet"]
```

Comment:
Supporting Spark and launch would be a really nice stretch goal, but not a dealbreaker